Sources/HipMobileUI.iOS/HipMobileUI.iOS.csproj

```diff
@@ -452,27 +452,21 @@
   <ItemGroup>
     <ImageAsset Include="Resources\MapAnnotations.xcassets\AppIcons.appiconset\Contents.json">
       <Visible>false</Visible>
-      <InProject>false</InProject>
```

Comment:
I wonder what this does? I see VS sometimes adds and sometimes removes these kinds of lines just on its own.